pkg/tests/ossm/operator/clusterwide_mode_test.go

```diff
@@ -305,6 +305,78 @@ spec:
 			})
 		})
 
+		t.NewSubTest("verify that namespaces without istio-enable label are not included to the SMMR list").Run(func(t test.TestHelper) {
+			t.LogStep("Create 5 member namespaces")
+			createMemberNamespaces(t, 5)
+
+			t.LogStep("Wait for SMMR to be Ready")
+			oc.WaitSMMRReady(t, meshNamespace)
+			t.Logf("Actual SMMR members: %s", meshNamespace)
+
+			t.LogStep("Remove istio-injection=enabled label from member-1 and member-3 namespaces")
+			for _, member := range []string{"member-0", "member-2", "member-4"} {
```

Comment:
The log step said that will be deleted the label from member-1 and member-3 but in this for it's been deleted member-0, member-2 and member-4

Reply:
Done